src/main/java/net/datafaker/transformations/sql/SqlTransformer.java

```diff
@@ -163,6 +176,23 @@ private String handleObjectInCollection(Object value) {
         return result.toString();
     }
 
+    private String handeObjectInMap(Object value) {
```

Comment:
```suggestion
    private String handleObjectInMap(Object value) {
```
seems like a misprint